extension-config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": "0.1.0",
+  "id": "@shopgate/ext-larger-product-image-on-list-view",
```

Comment:
only the github repo should have the `ext-` prefix, not the ID.
i also had to change the ID to `larger-product-image-on-list`because of some char limitations in the dev admin

Reply:
These changes are made